browser_history/__init__.py

```diff
@@ -77,3 +79,33 @@ def get_bookmarks():
             utils.logger.info("%s", e)
     output_object.bookmarks.sort()
     return output_object
+
+
+# keep everything lower-cased
+browser_aliases = {
+    "google-chrome": "chrome",
+    "chromehtml": "chrome",
+    "chromium-browser": "chromium",
+    "msedgehtm": "edge",
+    "operastable": "opera",
+    "firefoxurl": "firefox",
+}
```

Comment:
Having this in `__init__` isn't a good idea, it will pollute the namespace (it will show up in suggestions for `from browser_history import `).

Maybe define this and `default_browser()` in `utils` and import it here?

Reply:
I had written it in `utils` but moved it once I realized circular import was gonna be a problem. This is because `default_browser` uses `get_browsers` and `__init__.py` imports `utils.py`. To access `get_browsers` I would need to import `__init__.py` from `utils`. Circular import! But I'll see if I can circumvent the circular import.

Reply:
Oh okay! Maybe we could move `get_browsers` also to utils (and import it in `__init__`)?

Reply:
`get_browsers` uses `generic.Browser` so moving it to `utils` would still cause a circular import. It's also the reason why I left it there when I wrote it instead of moving it to `utils`. Circular import is a serious nuisance.

Reply:
@Samyak2 what is your stand on this? I am not able to move these functionalities to `utils.py` without introducing a circular import issue.